shared-bindings/microcontroller/Processor.c

```diff
@@ -93,9 +93,34 @@ const mp_obj_property_t mcu_processor_temperature_obj = {
     },
 };
 
+//| .. attribute:: unique id
+//|
+//|   Return the unique id (aka serial number) of the chip.
+//|   Returns a bytearray object. Use the CPython ``struct``
+//|   library to unpack the bytearray. Unpacked result is
+//|   a singleton (single item tuple).
+//|   _Note: ``struct`` is currently not available on the nRF port._
```

Comment:
I wouldn't mention struct here. People can do what they want with it afterwards.